lib/Serge/Sync/Plugin/TranslationService/crowdin.pm

```diff
@@ -34,15 +37,21 @@ sub validate_data {
 
     $self->{data}->{config_file} = subst_macros($self->{data}->{config_file});
     $self->{data}->{upload_translations} = subst_macros($self->{data}->{upload_translations});
+    $self->{data}->{import_duplicates} = subst_macros($self->{data}->{import_duplicates});
+    $self->{data}->{import_eq_suggestions} = subst_macros($self->{data}->{import_eq_suggestions});
+    $self->{data}->{auto_approve_imported} = subst_macros($self->{data}->{auto_approve_imported});
 
     die "'config_file' not defined" unless defined $self->{data}->{config_file};
     die "'config_file', which is set to '$self->{data}->{config_file}', does not point to a valid file.\n" unless -f $self->{data}->{config_file};
 
     $self->{data}->{upload_translations} = 1 unless defined $self->{data}->{upload_translations};
+    $self->{data}->{import_duplicates} = 0 unless defined $self->{data}->{import_duplicates};
+    $self->{data}->{import_eq_suggestions} = 0 unless defined $self->{data}->{import_eq_suggestions};
+    $self->{data}->{auto_approve_imported} = 0 unless defined $self->{data}->{auto_approve_imported};
 }
 
 sub run_crowdin_cli {
-    my ($self, $action, $langs, $capture) = @_;
+    my ($self, $action, $langs, $flags, $capture) = @_;
```

Comment:
See my comment below regarding avoiding the flags usage. The action in should be passed with all the flags.